internal/ethapi/api.go

```diff
@@ -1089,14 +1090,15 @@ func FormatLogs(logs []vm.StructLog) []StructLogRes {
 }
 
 // RPCMarshalHeader converts the given header to the RPC output .
-func RPCMarshalHeader(head *types.Header) map[string]interface{} {
+func RPCMarshalHeader(head *types.Header, _ consensus.Engine) map[string]interface{} {
+	miner := head.Coinbase // use the coinbase, not the validator author address for istanbul
```

Comment:
Shouldn't we have to add both? Author and FeeCurrencyAddress?

Reply:
Well, it's a decision we must take. @mcortesi what do you think?

Reply:
i would not add it for now. we have enough problems with not backward compatibility issues. (thus i guess being broken already, we could, but probably we need to check everywhere for consistency, not part of this pr though)